### DISCUSSION TITLE
Toggle Switch

### DISCUSSION THREAD

**Discussion by tonysip** [twilio-labs/paste#2615] [General] [answered]
Hi! I'm working on a feature that requires disabling/enabling within the console— i saw a lot of radio buttons with enable/disable action within the console. I'm looking at alternative options for this switch and ran into the in-progress "Toggle Switch" component. I'm wondering if I can get a sneak peek / screenshot of it so I explore this upcoming "Toggle Switch" component.

![Screen Shot 2022-08-17 at 10 45 33 AM](https://user-images.githubusercontent.com/11367707/185207159-e2e343ca-a3a4-4c4b-b121-190b48533cae.png)

Thank you!

**Comment by serifluous** [comment]
Hey @tonysip ! Yes, you can check it out here: https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=8687%3A56474 It's already published to the Paste Components Figma library, currently called "Toggle (dev)".

If you're looking for alternatives to Radio Groups, one main difference to keep in mind is that Toggles are meant to turn on/off a feature immediately, while Radio Groups are part of forms and don't trigger an action immediately (only on save).

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonysip 

The new Switch Component just landed in Paste. You can find the docs for the React component, ready to be used in your products, right here https://paste.twilio.design/components/switch

Thanks for your patience!
